### ISSUE TITLE
Move the icon behind the date?

### ISSUE THREAD

**Issue by mboevink** [sfsam/Itsycal#70]
Is there anyway to move the icon so that it sits behind the user specified time?
![capture d ecran 2017-08-19 a 15 20 18](https://user-images.githubusercontent.com/3157750/29483748-f5ffe6b4-84f1-11e7-9e25-3d7c086c53da.png)

**Comment by sfsam** [comment]
No, there isn't a way to do this. What would be the reason to do this?

**Comment by mboevink** [comment]
preference?

**Comment by julianschacherpp** [comment]
@mboevink What do you mean by moving the icon behind the time? Do you mean moving the icon of Itsycal behind the macOS system time?

**Comment by mboevink** [comment]
@julianschacherpp so I want itsycal to handle both the time and date item in my task bar, but I want the date box to sit behind the time so it aligns nicely with the edge.

![29483748-f5ffe6b4-84f1-11e7-9e25-3d7c086c53da](https://user-images.githubusercontent.com/3157750/37574704-f21d5af0-2b76-11e8-9693-e66d754beb2a.png)

**Comment by julianschacherpp** [comment]
@mboevink The implementation of time in the Itsycal Icon would be a good step into making it more unified and flush with the mac task bar. Having the option to display the Itsycal date without any  border or background and just having the plain text would be nice as well tho.
Regarding the position of the Itsycal icon: You can move it to the right of the system time, if that's what you mean.
